powerspikegg/computation_models/fetcher/BUILD

```diff
@@ -1,3 +1,5 @@
+package(default_visibility = ["//visibility:public"])
```

Comment:
I guess you won't depend on the `fetcher_test` script, so just put a visibility option onto the `fetcher` target :)